Automattic/simplenote-ios
# Simperium: Sync'ing Account Entities

### Details
- Updates Simperium to 1.2
- Enables Sync'ing of the Account bucket, backed by **SPJSONStorage**
- Implements the **EmailVerification** entity

Depends on [Simperium PR 606](https://github.com/Simperium/simperium-ios/pull/606/)
Ref #1086 

cc @eshurakov WDYT? (Feedback definitely more than welcomed!!)

---

### Test: Unverified Account
0. **Update your mobile-secrets first please!**
1. Checkout commit f4930532 (contains debug code!)
2. Log into an unverified account

- [ ] Verify that the legend `Email Verification` is never printed in your console

### Test: Verified Account
0. **Update your mobile-secrets first please!**
1. Checkout commit f4930532 (contains debug code!)
2. Log into an unverified account

- [ ] Verify that the legend `Email Verification` does get printed in your console

### Release
These changes do not require release notes.
